net/libfab/libfab.c

```diff
@@ -212,11 +247,129 @@ static int libfab_ma_stop(struct m0_net_transfer_mc *net, bool cancel)
 /**
  * Used as m0_net_xprt_ops::xo_ma_fini().
  */
-static void libfab_ma_fini(struct m0_net_transfer_mc *net)
+static void libfab_ma_fini(struct m0_net_transfer_mc *tm)
 {
 	/* TODO: 
 	 * Reverse the actions of libfab_ma_init() 
 	 * */
+
+	struct m0_net_end_point *net;
+	struct m0_fab__ep_param *ep;
+
+	M0_ENTRY();
+
+	m0_tl_for(m0_nep, &tm->ntm_end_points, net) {
+		ep = libfab_ep_net(net);
+		libfab_ep_res_free(ep->fep_ep_res);
+		libfab_ep_param_free(ep);
+		/* TODO: Remove entry from tm->ntm_end_points list */
+	} m0_tl_endfor;
+
+	M0_LEAVE();
+}
+
+/** Converts generic end-point to its libfabric structure. */
+static struct m0_fab__ep_param *libfab_ep_net(struct m0_net_end_point *net)
+{
+	return container_of(net, struct m0_fab__ep_param, fep_nep);
+}
+
+static int libfab_ep_find(const char *name, struct m0_net_transfer_mc *tm, 
```

Comment:
Please comment the parameter and return value.

If possible, it seems "bool" is enough for its return value.